CONTRIBUTING.md

```diff
@@ -2,185 +2,79 @@
-
-The `vendor/` update needs to be justified as part of the PR description,
-and must be verified by the reviewers and/or merger to always reference
-an existing upstream commit.
+Before sending code out for review, run all the tests for the whole tree to make sure the changes don't break other usage and keep the compatibility on upgrade. To make sure you are running the test suite exactly like we do you should install the CLI for [Drone CI](https://github.com/drone/drone) as we are using the server for continous testing, follow [these instructions](http://readme.drone.io/0.5/install/cli/). After that you can simply call `drone exec` within you working directory and it will try to run the test suite locally.
 
 ## Code review
 
-Changes to Gitea must be reviewed before they are accepted, no matter
-who makes the change even if an owners or a maintainer.  We use github's
-pull request workflow to do that and use [lgtm](http://lgtm.co) to ensure
-every PR is reviewed by at least 2 maintainers.
-
-Please try to make your pull request easy to review for us. Please read the
-["How to get faster PR reviews"](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/faster_reviews.md),
-guide, it got useful tips for any project you may want to contribute. See some
-of the points:
-
-- Make small pull requests. The smaller, the faster to review and the more
-  likely it will be merged soon.
-- Don't make changes unrelated to your PR. Maybe there are typos on some
-  comments, maybe refactoring would welcome on a function... but if that is not
-  related to you PR, please make *another* PR for that.
-- Split big pull requests in multiple. An incremental change will be faster to
-  review than a huge PR.
+Changes to Gitea must be reviewed before they are accepted, no matter who makes the change even if an owner or a maintainer. We use GitHub's pull request workflow to do that and we also use [LGTM](http://lgtm.co) to ensure every PR is reviewed by at least 2 maintainers.
 
 ## Sign your work
 
-The sign-off is a simple line at the end of the explanation for the
-patch. Your signature certifies that you wrote the patch or otherwise
-have the right to pass it on as an open-source patch. The rules are
-pretty simple: If you can certify [DCO](DCO), then you just add a line
-to every git commit message:
+The sign-off is a simple line at the end of the explanation for the patch. Your signature certifies that you wrote the patch or otherwise have the right to pass it on as an open-source patch. The rules are pretty simple: If you can certify [DCO](DCO), then you just add a line to every git commit message:
 
 ```
 Signed-off-by: Joe Smith <joe.smith@email.com>
 ```
 
-Please use your real name, we really dislike pseudonyms or anonymous
-contributions. We are in the opensource world without secrets. If you
-set your `user.name` and `user.email` git configs, you can sign your
-commit automatically with `git commit -s`.
-
-## Contributors
```

Comment:
Because you can't keep this file updated at all. Everybody who does only a single commit to this repo is automatically a contributor. Instead of that file I am linking to the GitHub graph which shows all contributors pretty properly.